suite-native/intl/src/en.ts

```diff
@@ -757,29 +757,39 @@ export const en = {
                 defaultPassphrase: 'Passphrase wallet #{index}',
             },
         },
-        deviceChecks: {
-            title: 'Device checks',
-            firmwareAuthenticityCheck: {
-                title: 'Turn off firmware authenticity check',
-                subtitle:
-                    'Firmware authenticity check is a crucial security feature. We strongly recommend keeping it turned on.',
+        advanced: {
+            title: 'Advanced',
+            goodAdvice: 'We strongly recommend leaving these checks turned on.',
+            authenticityChecks: {
                 buttonTurnOff: 'Turn off',
                 buttonTurnOn: 'Turn on',
                 buttonLearnMore: 'Learn more',
-
-                turnOffModal: {
-                    title: 'Turn off firmware authenticity check',
-                    content:
-                        'Trezor Support will never ask you to turn off the firmware revision check. This feature is designed to protect your security.',
-                    item1: 'Only if the device has passed the check before',
+                toastOn: 'Check turned on',
+                toastOff: 'Check turned off',
+                turnOff: {
+                    content: 'This feature is designed to protect your security.',
+                    item1: 'Only continue if the device has passed this check before',
                     item1Explanation:
-                        'Using an unverified device could result in the loss of funds.',
-                    item2: 'Only for testing and development',
+                        'Using an unverified device could compromise the security of your funds.',
+                    item2: 'Only use for testing and development',
                     item2Explanation:
-                        'These security checks should only be disabled for testing and development purposes.',
+                        'This security check should only be disabled for testing and development purposes.',
                     acknowledgement: 'I’ve read and understood the above',
+                    acknowledgementNote: 'Trezor Support will never ask you to turn this off.',
```

Comment:
🤔 
```suggestion
                    acknowledgementNote: 'Trezor Support will never ask you to turn this check off.',
```

Reply:
I don't think it's necessary there, on the button, there is only "Turn off" label too. I don't believe this would add some clarity.

Reply:
In my case adding the word "check" is moving "off" on the second line, without it it's on one line...

Reply:
For me, the sentence seems incomplete without "check". Up to you.